core/src/main/java/greencity/configuration/SecurityConfig.java

```diff
@@ -182,7 +187,8 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                     UBS_MANAG_LINK + "/delete-violation-from-order/{id}",
                     UBS_MANAG_LINK + "/delete-manual-payment/{id}",
                     UBS_MANAG_LINK + "/deleteCertificate/{code}",
-                    SUPER_ADMIN_LINK + "/**")
+                    SUPER_ADMIN_LINK + "/**",
+                    USER_AGREEMENT_LINK + "/**")
```

Comment:
Why do we need these configs twice?
[Here](https://github.com/ita-social-projects/GreenCityUBS/pull/1474/files#diff-05eb7be5959ff2ad17a6652cc076f4be0fb0dbdf640f016db9f572ded4297ee0R161) you have the same configs